package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hypergard",
-  "version": "5.2.0",
+  "version": "6.0.0",
```

Comment:
would you be opposed to adding an an `engines` section here to indicate which node + npm version to run?

```json
  "engines": {
    "node": ">=10.0.0",
    "npm": ">=6.0.0"
  },```

probably does it